ui/index.html

```diff
@@ -39,6 +39,11 @@
                             src="'partials/main/thGlobalTopNavPanel.html'">
                 </ng-include>
             </div>
+            <div uib-alert type="info" style="flex: none;" ng-show="serverChangedDelayed" ng-cloak>
+                <i class="fa fa-info-circle" aria-hidden="true"></i>
+                Treeherder has updated, to pick up the changes you can reload the page
+                <button ng-click="updateButtonClick()" class="btn btn-xs btn-danger">Reload?</button>
```

Comment:
This still flashes on the screen briefly, probably because it's part of index.html. Could you put it in an included template file, like thGlobalTopNavPanel.html above?

Also let's rename the button text from `Reload?` to just `Reload`